Index: app/src/main/java/com/example/android/pma_inteligentna_domacnost/Svetlo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/android/pma_inteligentna_domacnost/Svetlo.java	(date 1541886406773)
+++ app/src/main/java/com/example/android/pma_inteligentna_domacnost/Svetlo.java	(date 1541886406773)
@@ -1,12 +1,10 @@
 package com.example.android.pma_inteligentna_domacnost;
 
 import android.content.Intent;
-import android.content.SharedPreferences;
 import android.content.pm.ActivityInfo;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
-import android.widget.Switch;
 import android.widget.TextView;
 
 import com.google.gson.JsonObject;
@@ -23,25 +21,13 @@
 
 public class Svetlo extends AppCompatActivity {
 
-    String cislo_miestnosti;
-    String status;
     private TextView mTextViewResult;
-    public static final String SHARED_PREFS = "sharedPrefs";
-
-    // Switch 1
-    private Switch switch1;
-    public static final String SWITCH1 = "switch1";
-    private boolean switchOnOff1;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_svetlo);
         this.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LOCKED);
-        mTextViewResult = findViewById(R.id.result);
-        switch1 = findViewById(R.id.s1);
-        loadData();
-        updateViews();
     }
 
     public void submitOrder2(View view) {
@@ -55,27 +41,19 @@
     }
 
     public void submitOrder(View view){
-        loadData();
-        updateViews();
-
-        int ID = view.getId();
-        if (ID == R.id.s1) {
-            cislo_miestnosti = "1";
-            if (switchOnOff1) status = "on";
-            else status = "off";
-        }
-
-        OkHttpClient client = new OkHttpClient();
+        
+                OkHttpClient client = new OkHttpClient();
 
-        String url = "http://iot-python.eu-de.mybluemix.net/api/alarm";
+                String url = "http://iot-python.eu-de.mybluemix.net/api/alarm";
 
-        final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
-        JsonObject json = new JsonObject();
-        json.addProperty("senzor", "led");
-        json.addProperty("miestnost",cislo_miestnosti);
-        json.addProperty("status",status);
+                final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
+                JsonObject json = new JsonObject();
+                json.addProperty("senzor", "led");
+                json.addProperty("status",on_off);
+                json.addProperty("status",on_off);
+
 
-        RequestBody body = RequestBody.create(JSON,json.toString());
+                RequestBody body = RequestBody.create(JSON,json.toString());
 
                 Request request = new Request.Builder()
                         .url(url)
@@ -97,36 +75,17 @@
                             Svetlo.this.runOnUiThread(new Runnable() {
                                 @Override
                                 public void run() {
-                                    mTextViewResult.setText(myResponse);
-                                    saveData();
+                                    mTextViewResult.setText(sprava);
 
+                                    loadData();
+                                    updateViews();
 
                                 }
                             });
                         }
                     }
                 });
-
-
-    }
-
-
-
-    public void saveData() {
-        SharedPreferences sharedPreferences = getSharedPreferences(SHARED_PREFS, MODE_PRIVATE);
-        SharedPreferences.Editor editor = sharedPreferences.edit();
-
-        editor.putBoolean(SWITCH1, switch1.isChecked());
-        editor.apply();
-
-    }
-
-    public void loadData() {
-        SharedPreferences sharedPreferences = getSharedPreferences(SHARED_PREFS, MODE_PRIVATE);
-        switchOnOff1 = sharedPreferences.getBoolean(SWITCH1, false);
-    }
-
-    public void updateViews() {
-        switch1.setChecked(switchOnOff1);
+            
+        );
     }
 }
\ No newline at end of file
